baybe/recommenders/pure/bayesian/botorch.py

```diff
@@ -5,8 +5,9 @@
 from typing import Any, ClassVar
 
 import pandas as pd
-from attr.converters import optional
 from attrs import define, field
+from attrs.converters import optional as optional_c
```

Comment:
because there is also an `optional` in `attrs.validators`, in other places we call them `optional_c` and `optional_v` and while strictly not necessarily here, why not